_data/simple-icons.json

```diff
@@ -10065,6 +10065,18 @@
             "source": "https://www.ripple.com/media-kit/",
             "guidelines": "https://brand.ripple.com/article/brand-policy"
         },
+        {
+            "title": "RISC-V",
+            "hex": "FFC72C",
+            "aliases": {
+                "aka": [
+                    "RISC",
+                    "RISC Five"
+                ]
+            },
+            "source": "https://riscv.org/risc-v-logo/",
+            "guidelines": "https://riscv.org/about/risc-v-branding-guidelines/"
+        },
```

Comment:
```suggestion
            "title": "RISC-V",
            "hex": "283272",
            "source": "https://riscv.org/risc-v-logo/",
            "guidelines": "https://riscv.org/about/risc-v-branding-guidelines/"
        },
```

Those aliases are unnecessary.

And the color should be `#283272`. They have an example of monochrome image.

<img width="569" alt="image" src="https://github.com/simple-icons/simple-icons/assets/8186898/0d3509e9-638f-4b3d-99f0-664647ef1e34">

Reply:
![riscv](https://github.com/simple-icons/simple-icons/assets/35000807/84727144-b014-4494-ae46-41c5cc6bbf2a)

Updated and pushed the commit